impeller/entity/entity_unittests.cc

```diff
@@ -85,15 +85,19 @@ class TestPassDelegate final : public EntityPassDelegate {
 
  private:
   const std::optional<Rect> coverage_;
+  bool collapse_;
```

Comment:
Nit: Default it, or if it set in a ctor, `const bool`

Reply:
Done